src/sidebar/search-client.js

```diff
@@ -8,21 +12,28 @@ import { TinyEmitter } from 'tiny-emitter';
 export default class SearchClient extends TinyEmitter {
   /**
    * @param {Object} searchFn - Function for querying the search API
-   * @param {Object} opts - Search options
+   * @param {Object} options
+   *   @param {number} [options.chunkSize] - page size/number of annotations
+   *   per batch
+   *   @param {boolean} [options.separateReplies] - When `true`, request that
```

Comment:
This option is new, and needed because I can't inject the `features` service here.